mixnode/src/config/mod.rs

```diff
@@ -360,7 +360,8 @@ pub struct MixNode {
     id: String,
 
     /// Layer of this particular mixnode determining its position in the network.
-    layer: u64,
+    /// Before getting the actual layer from the validators, this is marked as None.
```

Comment:
As I asked before, do we even need it? (I'm not 100% sure myself, but I kinda don't see any reasons for it)

Reply:
Removed it.